.github/workflows/test.yml

```
@@ -0,0 +1,57 @@
+name: Test
+
+on:
+  pull_request:
+  push:
```

I would recommend changing this to the follwoing before merging:

```suggestion
on:
  pull_request:
    branches:
      - master
  push:
```

Otherwise it will build twice.